runtime/conditions/getter.go

```diff
@@ -83,6 +83,24 @@ func IsUnknown(from Getter, t string) bool {
 	return true
 }
 
+// IsReady is true if IsStalled and IsReconciling are False, and meta.ReadyCondition is True, otherwise it is false if
+// the condition is not True or if it does not exist (is nil).
+func IsReady(from Getter) bool {
+	return !IsStalled(from) && !IsReconciling(from) && IsTrue(from, meta.ReadyCondition)
```

Comment:
Why isn't this just `IsTrue(from, meta.ReadyCondition)` did we changed the behaviour of `Ready`?

Reply:
Because `Stalled` and `Reconciling` are successors to `Ready`, and should take precedence if they exist over `Ready`.

> For resources that have not adopted the recommended conditions described above [...]

https://github.com/kubernetes-sigs/cli-utils/tree/master/pkg/kstatus